gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+net-javacrumbs-shedlock-shedlock-spring = "net.javacrumbs.shedlock:shedlock-spring:5.2.0"
+nz-net-ultraq-thymeleaf-thymeleaf-layout-dialect = { module = "nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect" }
+org-apache-santuario-xmlsec = "org.apache.santuario:xmlsec:3.0.2"
+org-awaitility = "org.awaitility:awaitility:4.2.0"
+org-bouncycastle-bcprov-jdk15on = "org.bouncycastle:bcprov-jdk15on:1.70"
+org-flywaydb-flyway-core = "org.flywaydb:flyway-core:9.18.0"
+org-jetbrains-kotlin-kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+org-jetbrains-kotlin-kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-reflect" }
+org-jetbrains-kotlin-kotlin-stdlib-jdk8 = { module = "org.jetbrains.kotlin:kotlin-stdlib-jdk8" }
+org-jsoup = "org.jsoup:jsoup:1.16.1"
+org-postgresql = "org.postgresql:postgresql:42.6.0"
+org-springdoc-springdoc-openapi-starter-webmvc-api = { module = "org.springdoc:springdoc-openapi-starter-webmvc-api", version.ref = "org-springdoc" }
+org-springdoc-springdoc-openapi-starter-webmvc-ui = { module = "org.springdoc:springdoc-openapi-starter-webmvc-ui", version.ref = "org-springdoc" }
+org-springframework-boot-spring-boot-configuration-processor = { module = "org.springframework.boot:spring-boot-configuration-processor" }
+org-springframework-boot-spring-boot-devtools = { module = "org.springframework.boot:spring-boot-devtools" }
+org-springframework-boot-spring-boot-starter-actuator = { module = "org.springframework.boot:spring-boot-starter-actuator" }
+org-springframework-boot-spring-boot-starter-aop = { module = "org.springframework.boot:spring-boot-starter-aop" }
+org-springframework-boot-spring-boot-starter-data-jpa = { module = "org.springframework.boot:spring-boot-starter-data-jpa" }
+org-springframework-boot-spring-boot-starter-security = { module = "org.springframework.boot:spring-boot-starter-security" }
+org-springframework-boot-spring-boot-starter-test = { module = "org.springframework.boot:spring-boot-starter-test" }
+org-springframework-boot-spring-boot-starter-thymeleaf = { module = "org.springframework.boot:spring-boot-starter-thymeleaf" }
+org-springframework-boot-spring-boot-starter-validation = { module = "org.springframework.boot:spring-boot-starter-validation" }
+org-springframework-boot-spring-boot-starter-web = { module = "org.springframework.boot:spring-boot-starter-web" }
+org-springframework-boot-spring-boot-starter-webflux = { module = "org.springframework.boot:spring-boot-starter-webflux" }
+# @pin
+org-springframework-cloud-spring-cloud-starter-kubernetes-client-config = "org.springframework.cloud:spring-cloud-starter-kubernetes-client-config:2.1.3"
+org-springframework-data-spring-data-redis = "org.springframework.data:spring-data-redis:3.0.5"
+org-springframework-security-spring-security-test = { module = "org.springframework.security:spring-security-test" }
```

Comment:
If we do not use in the gradle file why do we need it here?

Reply:
It is still used in unit tests, line 123.